libraries/lib-datahandler/src/main/thrift/licenseinfo.thrift

```diff
@@ -59,25 +61,23 @@ struct LicenseInfoParsingResult {
     32: optional string version,
 }
 
-service LicenseInfoService {
+struct LicenseInfoFile {
```

Comment:
This allows us to return more information with one call. This way, the backend must only be called once for license info generation.